code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -1058,6 +1058,26 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	if(!target)
 		return 0 //wait where'd they go
 	clockwork_generals_invoked["sevtug"] = world.time + CLOCKWORK_GENERAL_COOLDOWN
+
+	// You can throw off the mind control if you're shielded
+	for(var/obj/item/weapon/implant/mindshield/M in target)
+		if(M.implanted)
+			M << "<span class='sevtug'>Lbhe anabobgf jvyy abg cebgrpg lbh \
+				sberire...</span>\n\
+				<span class='userdanger'>You feel like something tried to \
+				control you, but was unable to</span>"
+			qdel(M)
+
+			invoker << "<span class='sevtug'>Juvyr V fgvyy jnvg va gur \
+				Pryrfgvny Qreryvpg, zl graqevyf ner irel sentvyr. Fbzrguvat \
+				vagrecbfrq vgfrys orgjrra zr naq gur gnetrg, naq \
+				fnpevsvprq vgfrys.</span>\n\
```

Comment:
Nitpick: Seems kind of unfitting of Sevtug, who's generally a jerk to everyone and doesn't really take the whole situation seriously. Perhaps "Gurl chg fbzr zrgny va gurve urnq gb cebgrpg gurz. Vf guvf gur yngrfg snfuvba gurfr qnlf?" or something like that would work better?